README.md

```diff
@@ -38,55 +88,59 @@ By default, this package will look for your Apple Search Ads data in the `apple_
 config-version: 2
 
 vars:
-    apple_search_ads_schema: your_schema_name
-    apple_search_ads_database: your_database_name 
+  apple_search_ads__using_search_terms: true # enable if you want to include the search_term_report table and want search_term_report related metrics reported
 ```
 
-For additional configurations for the source models, visit the [Apple Search Ads source package](https://github.com/fivetran/dbt_apple_search_ads_source).
-
-
-### Change the Build Schema
-
-By default, this package will build the Apple Search Ads staging models within a schema titled (`<target_schema>` + `_stg_apple_search_ads`) and the final Apple Search Ads models within a schema titled (`<target_schema>` + `_apple_search_ads`) in your target database. If this is not where you would like your modeled Apple Search Ads data to be written to, add the following configuration to your `dbt_project.yml` file:
+### Change the build schema
+By default, this package builds the Apple Search Ads staging models within a schema titled (`<target_schema>` + `_apple_search_ads_source`) and your Apple Search Ads modeling models within a schema titled (`<target_schema>` + `_apple_search_ads`) in your destination. If this is not where you would like your Apple Search Ads data to be written to, add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
 models:
-    apple_search_ads:
-        +schema: my_new_schema_name # leave blank for just the target_schema
     apple_search_ads_source:
-        +schema: my_new_schema_name # leave blank for just the target_schema
+      +schema: my_new_schema_name # leave blank for just the target_schema
+    apple_search_ads:
+      +schema: my_new_schema_name # leave blank for just the target_schema
 ```
-## Contributions
+    
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
 
-Don't see a model or specific metric you would have liked to be included? Notice any bugs when installing and running the package? If so, we highly encourage and welcome contributions to this package! 
-Please create issues or open PRs against `master`. See [the Discourse post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) for information on how to contribute to a package.
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_apple_search_ads/blob/main/dbt_project.yml) variable declarations to see the expected names.
```

Comment:
This link seems to be broken

Reply:
This link seems to be working for me 🤔 (C&p'd the link and went to the branch and clicked directly from the README)

Reply:
Same as the source. It seems my computer was just acting a funk when I reviewed this PR.